src/status_im/ui/screens/home/views.cljs

```diff
@@ -96,7 +98,29 @@
                        (when-not search-filter
                          (re-frame/dispatch [:search/home-filter-changed ""])))
      :on-change      (fn [text]
-                       (re-frame/dispatch [:search/home-filter-changed text]))}]])
+                       (re-frame/dispatch [:search/home-filter-changed text])
+                       (when-not (seq chats)
+                         (debounce/debounce-and-dispatch [:new-chat/set-new-identity text] 300)))}]])
```

Comment:
do we clean it later? for example, if a user presses cancel? i mean **just** to make sure if we open creat new chat later input field will be empty

Reply:
It is cleared when we open `start a new chat` screen. But I will add now also clear here, so if we will use in other palces too, it will be clean.